v3/lib/class-acf-to-rest-api-acf-api.php

```diff
@@ -130,8 +132,17 @@ public function get_field_objects( $id ) {
 			$fields_tmp = array();
 
 			if ( function_exists( 'acf_get_field_groups' ) && function_exists( 'acf_get_fields' ) && function_exists( 'acf_extract_var' ) ) {
-				$field_groups = acf_get_field_groups( array( 'post_id' => $id ) );
-
+				if ( strpos( $id, 'user_' ) !== false ) {
```

Comment:
This condition never will match because of it:
https://github.com/airesvsg/acf-to-rest-api/pull/247/files#diff-2b1f470cce5f57d891c442678d8a5f6fR64